_includes/unscheduled-purchase-trustly.md

```diff
@@ -0,0 +1,81 @@
+## Unscheduled Purchase
+
+{% include alert.html type="warning" icon="warning" body="please note that this feature is only available through Payment Order." %}
+
+An `unscheduled purchase`, also called a Merchant Initiated Transaction (MIT),
+is a payment which uses a `paymentToken` generated through a previous payment in
+order to charge the same card at a later time. They are done by the merchant
+without the user being present.
+
+To use this, you need to make sure `directDebitEnabled` is `true` in the Trustly contract. After that you need to have the field `generateUnscheduledToken` set to `true` when creating an initial Purchase or Verify. Another important step is to make sure the `email` field is set, as this is a required parameter.
+
+{:.code-view-header}
+**Request**
+
+```http
```

Comment:
For me, it feels weird to document this as this is never relevant to the merchant

Reply:
Do you suggest removing the request from the documentation?

Reply:
Is this resolved? :)

Reply:
Yes, agreed that it wasn't relevant info for the merchant, so removed the request-response objects